demo/lightning/text-based/README.md

```diff
@@ -0,0 +1,57 @@
+# Parquet Demo Code
+The code examples in this directory demonstrate how dataflux can be used for text-based training alongside Pytorch-Lightning. To support this training, this example subclasses DataFluxIterableDataset for compatibility with parquet files from the HuggingFace FineWeb dataset.
+
+Please note that these demos are for educational and example purposes only, and have not been optimized for performance.
+
+## Single Node
+The single node exmaple code here can be run on your local workstation with a command such as the following:
+
+```
+# python3 ./model.py --project=test-project --bucket=my-fineweb-data --batch-size=100 --prefix="fineweb/sample/10BT"
+```
+
+This will perform a simple autoencoder training pass that leverages lightning to skip writing a large volume of boilerplate code.
+
+## Distributed
+The distributed demo provides a simple example of how the DatafluxIterableDataset can be extended to function seamlessly with distributed GKE runs facilitated by Lightning.
+
+To execute the demo-code locally and simulate the use of mulltiple nodes, use the following commands. 
+
+*Note: The number of batches processed by each node must be identicle across all nodes to avoid freezing/deadlock during DDP executions. In the demo this can be guaranteed by providing the `--limit-train-batches` argument with a value less than the minimum batch count for a given node.*
```

Comment:
Typo: identical

also I’m not sure if “node” is really the correct term here?

Reply:
replacing with process to avoid confusion.